scl/cisco/plugin.conf

```diff
@@ -45,9 +45,18 @@ block parser cisco-timestamp-parser(template()) {
         };
 
         if {
+            # timestamp from Cisco Unified Call Manager, example "Jun 14 11:57:27 PM.685 UTC"
+            # NOTE: drops msecs and timezone
+            filter { match('^[.*]?([A-Za-z]{3} [0-9 ]\d \d{2}:\d{2}:\d{2} ((AM)|(PM)))' value('1') flags(store-matches)); };
```

Comment:
Do we need to put AM and PM into capture groups? Isn't (AM|PM) enough?
Also should we unset additional capture groups, e.g. $4 as well?
Testing with the input message we will have these nv-pairs as well:
"4":"PM", 
"0":"Jun 14 11:57:27 PM"

Reply:
With the already merged ewmm() driver that excludes all numeric matches I think we can get rid of all unsets() everywhere. That's fragile anyway (e.g I broke it again).

Reply:
done in the recent push.

Reply:
That functionality is in format-ewmm template function, right?
Which is used in syslog-ng() destination driver currently.

What if somebody wants to use only cisco-parser()? Should cisco-parser give you these temporary variables?
Or I change the question, is it a problem for the user if cisco-parser unsets these matches?

I know that name-value pairs are not transferred automatically, it's a sort of reference question for developers how to write processing elements.